src/resources/js/components/EditPresentation.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+    <v-form
+        ref="form"
+        v-model="valid"
+        lazy-validation
+    >
+      <v-text-field
+            v-model="updatePresentationInfo.title"
+            :rules="titleRules"
+            label="Titre"
+      />
+
+      <v-datetime-picker
+        label="Date et Heure"
+        v-model="updatePresentationInfo.date"
+        clearText="annuler"
+      >
+        <template slot="dateIcon">
+          <v-icon>mdi-calendar</v-icon>
+        </template>
+        <template slot="timeIcon">
+          <v-icon>mdi-clock</v-icon>
+        </template>
+      </v-datetime-picker>
+
+      <v-btn
+            :disabled="!valid"
+            color="success"
+            class="mr-4"
+            @click="submitChange"
+            >
+            Appliquer
+      </v-btn>
+    </v-form>
+</template>
+
+<script>
+import dateFormat from 'dateformat';
+
+let alert = {};
+export default {
+  props: ['parentRefs'],
+  data() {
+    return {
+      valid: true,
+      titleRules: [
+        (v) => v === '' || (v && v.length <= 20) || 'Le titre doit contenir moins de 20 caractères',
```

Comment:
if title is empty message must be different

Reply:
Changed in 5cce542